src/Contao/View/Contao2BackendView/ButtonRenderer.php

```diff
@@ -320,6 +320,19 @@ private function buildCommand($command, $model, $previous, $next, $isCircularRef
             return \trim($html);
         }
 
+        if ($buttonEvent->isDisabled() && $command instanceof ToggleCommandInterface) {
```

Comment:
Isn't all of this exactly the same as line 323-335 except for adding a title and cursor to it? If so, we should change the implementation below instead of adding yet another special case for togglers.

Reply:
No, is completely different:
in the lower case the standard buttons disabled e.g. the blue arrow for moving - this becomes grey with the attached underline.

Currently the icons disappear from the list when readonly - this is a bad usability!

With my change the icons stay in the list but don't have an onClick-event anymore and on hover the cursor changes and the title has a prefix (Disabled).

Please note that only the toggle icons have (or can have) completely different file names and not iconname.svg for enabled and iconname_.svg for disabled like the Contao standard...

Reply:
fixed with @discordier

Reply:
**Note:**
@discordier & @baumannsven 

Currently the readonly is only checked when the button is created.

If the option is set by the admin for an existing list, it is still possible to toggle...